samples/compression/lz4/sample.yaml

```diff
@@ -12,6 +12,8 @@ common:
     type: one_line
     regex:
       - "Validation done. (.*)"
+  modules:
```

Comment:
According to the new docs, this sample should not be in tree, or? (same for all others you modified here)

Reply:
correct, but this require agreement first and a period of time giving module owner a chance to move things around. Doing this in the PR will block progress.